.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,60 @@
+name: Commitlint
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+  merge_group:
+    branches:
+      - main
+    types:
+      - checks_requested
+jobs:
+  commit-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install conventional-commit linter
+        run: npm install @commitlint/config-conventional @commitlint/cli
+
+      # Default config does not allow sentence case
+      # https://github.com/conventional-changelog/commitlint/blob/master/@commitlint/config-conventional/index.js
+      - name: Add conventional commits config and enable sentence case
```

Comment:
I would like to see the configuration provided directly in the repo. It would allow local linting.